cterasdk/edge/drive.py

```diff
@@ -12,10 +12,19 @@ class Drive(BaseCommand):
     def get(self, name=None):
         """
         Get Drive. If a drive name was not passed as an argument, a list of all drives will be retrieved
+
         :param str,optional name: Name of the drive
         """
         return self._gateway.get('/config/storage/disks' + ('' if name is None else ('/' + name)))
 
+    def get_status(self, name=None):
```

Comment:
Just would like to ensure consistency of method naming across the SDK modules. For instance, in the network module I named the method to be `status`. The client can then invoke `filer.network.status()`. We should either rename `status` to `get_status` or vice versa.

Reply:
I've aligned the names to "get_configuration", "get_status" and "modify"